rootfs/etc/confd/templates/fluent.conf.tmpl

```diff
@@ -193,22 +193,22 @@
   buffer_queue_full_action drop_oldest_chunk
   # Set the chunk limit conservatively to avoid exceeding the GCL limit
   # of 10MiB per write request.
-  buffer_chunk_limit 2M
+  buffer_chunk_limit {{getv "/fluentd/out/kubesys/buffer/chunk/limit"}}
```

Comment:
Could we move the default settings like `2M` here to `ENV` steps of our Dockerfile?
```
ENV FLUENTD_OUT_KUBESYS_BUFFER_CHUNK_LIMIT=2M
...
```